app/services/podcast_service.rb

```diff
@@ -10,7 +10,9 @@ def conn
   end
 
   def get_json(uri)
-    response = conn.get(uri)
-    JSON.parse(response.body, symbolize_names: true)
+    Rails.cache.fetch(uri) do
```

Comment:
Lol I see what you mean now when you said maybe too lines of code but this looks great!